app/src/main/java/com/android/unio/model/map/nominatim/NominatimLocationResponse.kt

```diff
@@ -0,0 +1,21 @@
+package com.android.unio.model.map.nominatim
+
+import com.google.gson.annotations.SerializedName
+
+data class NominatimLocationResponse(
```

Comment:
Even though relatively self explanatory, some documentation is still welcome here too